src/sentry/incidents/models/alert_rule.py

```diff
@@ -223,6 +223,20 @@ def conditionally_subscribe_project_to_alert_rules(
             )
         return []
 
+    def get_for_metrics(
+        self, organization: Organization, metric_mris: list[str]
```

Comment:
Since span metrics are defined on a per-project level, shouldn't this take project instead of organization as a parameter?

Reply:
After our conversation, I have decided to keep organization as an argument since we filter only on it later in the method